benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -17,6 +17,9 @@
 
 #include "ppx/graphics_util.h"
 
+// Use this to disable sphere rendering completely, including loading
+//#define DISABLE_SPHERES
```

Comment:
this is used to speed up the loading time for bandwidth test since in this case only full screen quads are needed.